src/test_l1_dsVideoDevice.c

```diff
@@ -94,6 +95,21 @@ static int gTestID = 1;
     }\
 }\
 
+#define CHECK_FOR_EXTENDED_ERROR_CODE( result, enhanced, old )\
```

Comment:
#defines have GLOBAL Scope, you'll be re-using the same macro in all files..

therefore has to be module specific

`DS_VIDEO_DEVICE_CHECK_FOR_EXTEND_ERROR_CODE`

it's only working by luck.